docs/software/koji-workflow.md

```diff
@@ -47,53 +47,137 @@ The results of a buildArch task. Their metadata are recorded in the Koji databas
 **Repository**  
 A yum repository created from the contents of a tag at a specific point in time. By default, the yum repository will contain all successful, non-blocked builds in the tag, plus all RPMs in the external repositories for the tag.
 
-Using Koji
-----------
 
-### Required Software
+Obtaining Access
+----------------
+
+Building OSG packages in Koji requires these privileges:
+
+- access to the OSG subversion repository at https://vdt.cs.wisc.edu/svn
+- access to a login node at UW Comp Sci such as `moria.cs.wisc.edu`
+- access to the Koji service via a grid user certificate
+
+See the [user certificates document](https://opensciencegrid.org/docs/security/user-certs/)
+for information about how to get a user certificate.
+
+Open a Freshdesk ticket with the subject "Requesting access to Koji" with the following information:
+- top 3 username choices for the login node and SVN
+  (8 characters max, no punctuation)
```

Comment:
No punctuation as in no unicode accent marks over the a's in your name?

Or is it more like `^[A-Za-z_]\w{0,7}$`

Reply:
Technically the latter.
While saying "your username choice must match `/^[A-Za-z_]\w{0,7}$/`" is a nice shibboleth for new contributors, I don't think we wanna be that way :-)

Reply:
"As your first test, you must pass through the new user regex!"  ;)